src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -275,13 +241,14 @@ public void render(Graphics g)
         currentAnimation = null;
 
         // The animation is over; remove the corresponding event and notify the controller.
-        mapController.animationEnded(eventsToAnimate.poll(), eventsToAnimate.isEmpty());
+        if( notifyOnAnimEnd ) // ...but only if we're animating events
+          mapController.animationEnded(eventsToAnimate.poll(), eventsToAnimate.isEmpty());
 
         // Get the next event animation if one exists.
         loadNextEventAnimation();
       }
     }
-    else if( getCurrentGameMenu() == null )
+    if( getCurrentGameMenu() == null )
```

Comment:
Hm. Won't this draw the cursor even during animations? I kinda like having the cursor disappear whenever animations happen. Just another cue that the user can't use the cursor. Guess it probably wouldn't be the end of the world.

Reply:
Yeah.